internal/sync.go

```diff
@@ -680,24 +676,37 @@ func DoSync(ctx context.Context, cfg *config.Config) error {
 
 	c := New(cfg, awsClient, googleClient)
 
+
+	// TODO: by default, use sync method=users_groups. Skip the other method for now
+	// until we can do group and user name match.
 	log.WithField("sync_method", cfg.SyncMethod).Info("syncing")
-	if cfg.SyncMethod == config.DefaultSyncMethod {
-		err = c.SyncGroupsUsers(cfg.GroupMatch)
-		if err != nil {
-			return err
-		}
-	} else {
-		err = c.SyncUsers(cfg.UserMatch)
-		if err != nil {
-			return err
-		}
+	err = c.SyncUsers(cfg.UserMatch)
+	if err != nil {
+		return err
+	}
 
-		err = c.SyncGroups(cfg.GroupMatch)
-		if err != nil {
-			return err
-		}
+	err = c.SyncGroups(cfg.GroupMatch)
+	if err != nil {
+		return err
 	}
 
+	// if cfg.SyncMethod == config.DefaultSyncMethod {
```

Comment:
should all of this really be commented? maybe just remove it then?

Reply:
Later we probably gonna use the other sync method? Not sure about it...